src/migrations/00031-drop-supervisor-release.sql

```diff
@@ -0,0 +1,7 @@
+DROP TABLE "supervisor release";
```

Comment:
This needs to be the last bit, after all the things that depend on it have been dropped

Reply:
updated.